frameworks/hdfs/src/main/dist/krb5.conf

```diff
@@ -1,14 +1,14 @@
 [libdefaults]
-default_realm = LOCAL
-dns_lookup_realm = true
-dns_lookup_kdc = true
-udp_preference_limit = 1
+  default_realm = {{KERBEROS_REALM}}
```

Comment:
Don't we need to add references to this file in the environment too?
```
-Djava.security.krb5.conf=$MESOS_SANDBOX/{{KAFKA_VERSION_PATH}}/config/krb5.conf`
```
(taken from Kafka)

Reply:
OK. Scratch that. THat's what's tracked in that other JIRA ticket, right? https://jira.mesosphere.com/browse/INFINITY-2726